x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/search/SearchApplicationIndexService.java

```diff
@@ -163,6 +163,10 @@ private static XContentBuilder getIndexMappings() {
                     builder.field("type", "keyword");
                     builder.endObject();
 
+                    builder.startObject(SearchApplication.INDICES_FIELD.getPreferredName());
```

Comment:
Why are we adding this back to the system index? Isn't this the opposite of what we want to achieve here?

Reply:
Thanks for catching this. I missed this part.

Reply:
Updated